crates/allocator/Cargo.toml

```diff
@@ -15,8 +15,10 @@ categories = ["no-std", "embedded"]
 include = ["Cargo.toml", "src/**/*.rs", "README.md", "LICENSE"]
 
 [dependencies]
-wee_alloc = { version = "0.4", default-features = false }
+wee_alloc = { version = "0.4", default-features = false, optional = true }
 
 [features]
 default = ["std"]
 std = []
+bump-allocator = []
+wee-allocator = ["wee_alloc"]
```

Comment:
We actually don't need 2 crate features if we declare one as default. If this experiment is successful we might declare the `bump-allocator` as the default and have `wee-allocator` as opt-in feature to use that allocator instead for contracts that actually need it.

Reply:
Yeah, I figured this would something to simplify later if we decide that the bump allocator is worth having

Reply:
Even the first iteration reduced Wasm file sizes by 1.2-1.3kB each so it is already a good win there. I am very sure we will see the `bump_allocator` being our default allocator soon.